gradio/external.py

```diff
@@ -4,7 +4,7 @@
 
 import requests
 
-from gradio import inputs, outputs
+from gradio import components
```

Comment:
Should not we test gradio.inputs and gradio.outputs as well? We are still supporting those components.

Reply:
You're right, we should still test `gradio.inputs` and `gradio.outputs`. All I'm doing here is just updating `external.py` to use `components` instead of `inputs` and `outputs`